src/config.c

```diff
@@ -3175,6 +3175,7 @@ standardConfig static_configs[] = {
     createSpecialConfig("replicaof", "slaveof", IMMUTABLE_CONFIG | MULTI_ARG_CONFIG, setConfigReplicaOfOption, getConfigReplicaOfOption, rewriteConfigReplicaOfOption, NULL),
     createSpecialConfig("latency-tracking-info-percentiles", NULL, MODIFIABLE_CONFIG | MULTI_ARG_CONFIG, setConfigLatencyTrackingInfoPercentilesOutputOption, getConfigLatencyTrackingInfoPercentilesOutputOption, rewriteConfigLatencyTrackingInfoPercentilesOutputOption, NULL),
 
+    createULongLongConfig("maxpostnotificationsjobs", NULL, MODIFIABLE_CONFIG, 0, ULLONG_MAX, server.max_post_notifications_jobs, 1000, INTEGER_CONFIG, NULL, NULL),
```

Comment:
Not only a module that misbehaving can cause an infinite loop, it might be a combination of two modules that running alone will behave just right but together will cause an infinite loop. This is why I thought to protect it on the Redis level and print a warning.
If we allow module to set it then what happened if multiple modules set it? We will take the highest value?

Reply:
yes, we can take the higher value, or the sum...
not sure about the details, but i'm pretty sure it shouldn't be a user config.

Reply:
@oranagra I removed the configuration value and created a module API to set it. Default is 1000. I also updated the top comment.